app/jobs/appointment_notification/worker.rb

```diff
@@ -1,117 +1,35 @@
 class AppointmentNotification::Worker
   include Sidekiq::Worker
+  include Memery
+  APPOINTMENT_REMINDERS_CHANNEL = CountryConfig.current[:appointment_reminders_channel].constantize
```

Comment:
As we discussed, let's make the worker _even_ thinner, and push this down to the dispatch service.